lib/cosmos/api/tlm_api.rb

```diff
@@ -167,51 +168,19 @@ def tlm_variable(*args, scope: $cosmos_scope, token: $cosmos_token)
     # @param args The args must either be a string followed by a value or
     #   three strings followed by a value (see the calling style in the
     #   description).
-    def set_tlm(*args, scope: $cosmos_scope, token: $cosmos_token)
+    def set_tlm(*args, type: :CONVERTED, scope: $cosmos_scope, token: $cosmos_token)
```

Comment:
The way set_tlm used to work is that if you set a value it would affect the FORMATTED and WITH_UNITS because we were directly writing the value into the packet. It now only affects the CVT type you specify. This could have implications for existing scripts which use set_tlm but not sure how else to handle it.